### PR TITLE
Separates builtin commands from user defined 

### LINKED ISSUE #72: Show user defined tasks in a different section
When showing the command line help user defined tasks are mixed with default idephix tasks such ad "selfupdate", "list", "init-id-file".

Showing user defined tasks in a separate section could add clarity, what do you guys think?

### PR DESCRIPTION
**Pull request by micheleorselli**
fixes #72. Tried a couple of different ways of doing that, ended up overwriting some console classes, not sure it's the right way but it works

### CODE REVIEW

File: src/Idephix/Console/Application.php [hunk 1]
```diff
@@ -4,16 +4,22 @@
 
 use Symfony\Component\Console\Application as BaseApplication;
 use Symfony\Component\Console\Input\InputOption;
+use Idephix\Console\Command\HelpCommand;
```
Comment (ftassi) [inline src/Idephix/Console/Application.php]: Is this class needed? Our `\Idephix\Console\Command\HelpCommand` seems identical to `\Symfony\Component\Console\Command\HelpCommand`, isn't it?

File: src/Idephix/Console/Descriptor/TextDescriptor.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    }
+
+    /**
+     * Formats input option/argument default value.
+     *
+     * @param mixed $default
+     *
+     * @return string
+     */
+    private function formatDefaultValue($default)
```
Comment (ftassi) [inline src/Idephix/Console/Descriptor/TextDescriptor.php]: This seems to be unused

File: src/Idephix/Idephix.php [hunk 3]
```diff
@@ -16,8 +16,8 @@
 use Symfony\Component\Console\Output\ConsoleOutput;
 use Idephix\SSH\SshClient;
 use Idephix\Extension\IdephixAwareInterface;
-use Idephix\Task\SelfUpdate\SelfUpdate;
-use Idephix\Task\InitIdxFile;
+use Idephix\Task\Builtin\SelfUpdate;
+use Idephix\Task\Builtin\InitIdxFile;
```
Comment (ftassi) [inline src/Idephix/Idephix.php]: 👍 like the new namespace

File: src/Idephix/Console/Application.php (hunk 1 above)
Comment (micheleorselli) [inline src/Idephix/Console/Application.php]: It's not needed, going to remove that

File: src/Idephix/Console/Descriptor/TextDescriptor.php (hunk 2 above)
Comment (micheleorselli) [inline src/Idephix/Console/Descriptor/TextDescriptor.php]: yup

**Comment by ftassi** [comment]
👍

**Pull request merged**